2-networks/README.md

```diff
@@ -12,7 +12,8 @@ The purpose of this step is to setup shared VPCs with default DNS, NAT, Private
 ### Setup to run via Cloud Build
 1. Clone repo `gcloud source repos clone gcp-networks --project=YOUR_CLOUD_BUILD_PROJECT_ID`
 1. Change freshly cloned repo and change to non master branch `git checkout -b plan`
-1. Copy contents of foundation to new repo `cp ../terraform-example-foundation/2-networks/* .`
+1. Copy contents of foundation to new repo `cp ../terraform-example-foundation/2-networks/* .` (modify accordingly based on your current directory)
```

Comment:
I guess it's `cp -R ../terraform-example-foundation/2-networks/* .` here to include the module folder